core/modules/css/KvInterfaceImplMySql.cc

```diff
@@ -568,9 +568,11 @@ std::string KvInterfaceImplMySql::dumpKV() {
         if (row[1].first == nullptr or *row[1].first == '\0') {
             result += "\\N";
         } else {
+/*
             if (strchr(row[1].first, '\n') != nullptr) {
                 throw CssError("KvInterfaceImplMySql::dumpKV - value contains newline");
             }
+*/
```

Comment:
I don't think the output of the command is meant to be parsed or used anywhere beyond the usual human-driven debugging process. I've searched all potential uses of the command within the package, and I found no single use of it.

A problem here is that the original code crashed when attempting to dump table schemas loaded into CSS by the qserv-data-load.py tool.